nodeJS/express/views.md

```diff
@@ -161,11 +161,30 @@ Then, modify the previously created `index.ejs` to `include` the `navbar` with t
 </html>
 ```
 
-This will include the navbar with links in `index.ejs`. Reusable templates can be used to include headers and footers in all of your pages.
+This will include the navbar with links in `index.ejs`. Run your server and see! Reusable templates can be used to include headers and footers in all of your pages.
 
 Note the use of the raw output tag `<%-` with the `include` which is used to avoid double-escaping the HTML output.
 
-Let's use `includes` to dynamically render a list of variables a different way. Add `const users = ["Bob", "Bill", "Biff"];` to `app.js` just below the `links` variable. Then create a new view called `user.ejs` in the `views` directory and add the following `<li><%= user %></li>` line. Finally add the following block to `index.ejs`:
+Let's use `include` to dynamically render a list of variables a different way. In `app.js`, add the following `users` array just below the `links` variable, and pass it to the `render` function when rendering `index`:
+
+```javascript
+// app.js
+
+const links = [...];
```

Comment:
We should either have the full links thing like before, or have a comment with the omission of something the keep as before. In this scenario, keeping the links array in full as before makes more sense to me. While `[...]` will probably make sense to many, I think it'd require a little thought time to realise, as opposed to just reading it straight up.
```suggestion
const links = [
  { href: "/", text: "Home" },
  { href: "about", text: "About" },
];
```

Reply:
That sounds good, I've now changed to use the full `links` variable.